CONTRIBUTING.md

```diff
@@ -1,3 +1,3 @@
 Any contribution that you make to this repository will
-be under the BSD license 2.0, as dictated by that
+be under the BSD 3 Clause license, as dictated by that
```

Comment:
The linter isn't happy with this, so I've reverted it. AFAIK, "BSD license 2.0" is a valid alias for the 3 clause license. We can change this later pending the outcome of https://github.com/ament/ament_lint/pull/209.